07/05_ip_adresy.py

```diff
@@ -0,0 +1,13 @@
+import re
+
+ip_adresa = input("Zadejte IP adresu: ")
+
+rv = re.compile(r"([1-2]?\d?\d\.){3}[1-2]?\d?\d")
```

Comment:
```suggestion
rv = re.compile(r"([12]?\d?\d\.){3}[12]?\d?\d")
```
Mělo byt fungovat i takto. Celkově vynikající!